multisig/src/lib.rs

```diff
@@ -201,7 +201,18 @@ impl MultiSigContract {
                     }
                 }
                 MultiSigRequestAction::DeleteKey { public_key } => {
-                    promise.delete_key(public_key.into())
+                    let pk:PublicKey = public_key.into();
+                    // delete outstanding requests by public_key
```

Comment:
This action can be used on other `receiver_id`. Also the key might not be part of the multi-key. Just need to be careful there

Reply:
> This action can be used on other `receiver_id`. Also the key might not be part of the multi-key. Just need to be careful there

Wondering if we should implement: https://github.com/near/core-contracts/issues/86 ?